src/udev/udev-node.c

```diff
@@ -423,6 +423,85 @@ static int node_get_current(const char *slink, int dirfd, char **ret_id, int *re
         return 0;
 }
 
+static int link_update_diskseq(sd_device *dev, const char *slink, bool add) {
+        _cleanup_free_ char *buf = NULL;
+        const char *fname, *diskseq;
+        int r;
+
+        assert(dev);
+        assert(slink);
+
+        if (!device_in_subsystem(dev, "block"))
+                return 0;
+
```

Comment:
there really should be a comment here, explaining what this function does, this is so not clear to me